.github/workflows/build_and_test.yml

```diff
@@ -133,6 +118,77 @@ jobs:
           if [[ "$(docker images -q integritee-worker-${{ env.IMAGE_SUFFIX }} 2> /dev/null)" != "" ]]; then
               docker image rmi --force integritee-worker-${{ env.IMAGE_SUFFIX }} 2>/dev/null
           fi
+          docker images --all
+
+    build-client:
+      runs-on: ${{ matrix.host }}
+      strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - flavor_id: sidechain
+            mode: sidechain
+            host: integritee-builder-sgx
+            sgx_mode: HW
+            additional_features: dcap
+          - flavor_id: offchain-worker
+            mode: offchain-worker
+            host: integritee-builder-sgx
+            sgx_mode: HW
+            additional_features: dcap
+          - flavor_id: teeracle
+            mode: teeracle
+            host: integritee-builder-sgx
+            sgx_mode: HW
+            additional_features: dcap
+          - flavor_id: sidechain-evm
+            mode: sidechain
+            additional_features: evm,dcap
+            host: integritee-builder-sgx
+            sgx_mode: HW
```

Comment:
I know that these are not strictly necessary for the client, but later in the integration tests and in the release process, we refer to the generated docker images by `IMAGE_SUFFIX` and I didn't want to change anything there for now.